src/signing.c

```diff
@@ -53,6 +57,7 @@ void aws_crt_signing_config_aws_set_region(
     size_t region_length) {
     struct aws_byte_buf input = aws_byte_buf_from_array(region, region_length);
     aws_byte_buf_init_copy(&signing_config->region, aws_crt_allocator(), &input);
+    signing_config->config.region = aws_byte_cursor_from_buf(&signing_config->region);
```

Comment:
memory leak if someone calls this more than once
the previous contents of the byte-buf are never freed

this probably applies to all the string setting functions...

Reply:
Oh shit, good catch.